### PR TITLE
add a new module to configure mdn on interface.

### PR DESCRIPTION
**Pull request by yuandongx**
##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

lib/ansible/modules/network/cloudengine/ce_mdn_interface.py
### test case
```
  - name: "test the correct lldpenable"
    ce_mdn_interface:
      lldpenable: enabled
      state: present

```
### test log info
```

changed: [10.190.121.125] => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python"
    },
    "changed": true,
    "end_state": [
        {
            "lldpenable": "enabled"
        }
    ],
    "existing": [
        {
            "lldpenable": "disabled"
        }
    ],
    "invocation": {
        "module_args": {
            "ifname": null,
            "lldpenable": "enabled",
            "mdnstatus": null
        }
    },
    "proposed": {
        "lldpenable": "enabled"
    },
    "updates": [
        [
            "lldp enable"
        ]
    ]
}
META: ran handlers
META: ran handlers

PLAY RECAP ******************************************************************************************************************************************************
10.190.121.125             : ok=1    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
```
### show on device
```
[~8850_130.12-100GE1/0/39]dis cu | in lldp
lldp enable
[~8850_130.12-100GE1/0/39]

```
### test case
```

  - name: "test the correct mdnstatus"
    ce_mdn_interface:
      mdnstatus: rxOnly
      ifname: 100GE1/0/39
      state: present
```
### test log info
```
changed: [10.190.121.125] => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python"
    },
    "changed": true,
    "end_state": [
        {
            "lldpenable": "enabled"
        },
        {
            "ifname": "100GE1/0/39",
            "mdnstatus": "rxOnly"
        }
    ],
    "existing": [
        {
            "lldpenable": "enabled"
        },
        {
            "ifname": "100GE1/0/39",
            "mdnstatus": "default"
        }
    ],
    "invocation": {
        "module_args": {
            "ifname": "100GE1/0/39",
            "lldpenable": null,
            "mdnstatus": "rxOnly"
        }
    },
    "proposed": {
        "ifname": "100GE1/0/39",
        "mdnstatus": "rxOnly"
    },
    "updates": [
        [
            "interface 100GE1/0/39",
            "lldp mdn enable"
        ]
    ]
}
META: ran handlers
META: ran handlers

PLAY RECAP ******************************************************************************************************************************************************
10.190.121.125             : ok=1    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
```
### show on device
```
[~8850_130.12-100GE1/0/39]di th
#
interface 100GE1/0/39
 eth-trunk 0
 lacp priority 10
 lldp mdn enable
#
return
[~8850_130.12-100GE1/0/39]

```

### CODE REVIEW

File: lib/ansible/modules/network/cloudengine/ce_mdn_interface.py [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+    type: str
+    choices: ['rxOnly', 'disabled']
+  ifname:
+    description:
+      - Interface name.
+    type: str
+  state:
+    description:
+      - Manage the state of the resource.
+    required: false
```
Comment (yuandongxx) [inline lib/ansible/modules/network/cloudengine/ce_mdn_interface.py]:
```suggestion
```

**Review by yuandongxx — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/network/cloudengine/ce_mdn_interface.py (hunk 1 above)
Comment (yuandongx) [reply lib/ansible/modules/network/cloudengine/ce_mdn_interface.py]: Done.

**Comment by xuxiaowei0512** [comment]
ready_for_review

**Review by yuandongxx — APPROVED** [review]

**Comment by hwDCN** [comment]
shipit

**Comment by hwDCN** [comment]
shipit

**Pull request merged**